test/support/models.rb

```diff
@@ -215,7 +215,7 @@ class ValidatingUser < User
     only_integer: true
   validates_length_of :name, maximum: 25
   validates_length_of :description, maximum: 50
-  validates_length_of :action, maximum: 10, tokenizer: lambda { |str| str.scan(/\w+/) }
+  validates_length_of :words_in_action, maximum: 10
```

Comment:
In Rails 5 you don't need to define that validation, so just remove it for Raisl 5